src/web.cpp

```diff
@@ -932,23 +932,23 @@ void handleRoot() {
         const char* ethMac = "ethMac";
         const char* ethSpd = "ethSpd";
         const char* ethIp = "ethIp";
-        const char* etchMask = "etchMask";
+        const char* ethMask = "ethMask";
         const char* ethGate = "ethGate";
         if (ConfigSettings.connectedEther) {
             doc[connectedEther] = yes;
             doc[ethConnection] = "Connected";
             doc[ethMac] = ETH.macAddress();
             doc[ethSpd] = String(ETH.linkSpeed()) + String(" Mbps");
             doc[ethIp] = ETH.localIP().toString();
-            doc[etchMask] = ETH.subnetMask().toString();
+            doc[ethMask] = ETH.subnetMask().toString();
```

Comment:
You also need to rename "etchMask" -> "ethMask" in the PAGE_ROOT.html file
![image](https://github.com/smlight-dev/slzb-06-firmware/assets/31830530/0c695e11-5435-404c-ae32-dee29abed903)

Reply:
Yeah. It's part of this commit.

Reply:
Thanks for these improvements